content-review/impl/compilatio/src/main/java/org/sakaiproject/contentreview/compilatio/CompilatioReviewServiceImpl.java

```diff
@@ -716,6 +716,21 @@ public String getIconUrlforScore(Long score) {
 	}
 
 	@Override
+	public String getIconCssClassforScore(Long score) {
+		if (score.equals(Long.valueOf(0))) {
```

Comment:
Could we switch the code a little (as @jonespm suggested switch to primatives) and make these:

```
if (score == 0) {
  return ...
} else if ( score < 25) {
  return ...
} else if (score < 50) {
  return ...
} else if (score < 75) {
```
As it's much easier to read the code as long as the values only ever increase.